src/vcs/git.rs

```diff
@@ -72,16 +72,17 @@ impl<'repo> GitRepository {
     /// let repo = GitRepository::new("./data/git-golden").unwrap();
     /// let browser = GitBrowser::new(&repo).unwrap();
     ///
-    /// let branches = browser.list_branches();
+    /// let mut branches = browser.list_branches().unwrap();
+    /// branches.sort();
```

Comment:
:eyes: Sneaky!

Reply:
It ain't a `SortedVec` ;P

But if we do need some sort of ordering then we can create an issue. I think this is perfectly justified for testing purposes :)

Reply:
Absolutely.